include/versions.h

```diff
@@ -55,4 +55,11 @@
 #define DEBUG_ASSETS 0
 #endif
 
+// Variables may be unused in retail versions but used in debug versions
+#if DEBUG_FEATURES
+#define UNUSED_NDEBUG
+#else
+#define UNUSED_NDEBUG UNUSED
+#endif
+
```

Comment:
perhaps include versions.h in attributes.h and move this define there? It seems more attribute-related than version-related to me